src/main/java/mekanism/common/tile/TileEntityTeleporter.java

```diff
@@ -1,14 +1,10 @@
 package mekanism.common.tile;
 
-import it.unimi.dsi.fastutil.longs.Long2ObjectMap;
-import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import it.unimi.dsi.fastutil.objects.ObjectOpenHashSet;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+
+import java.util.*;
```

Comment:
We don't use wildcard imports

Reply:
Ugh. IntelliJ kept doing that. :(

Reply:
This is the code formatting stylesheet we use with IntelliJ https://github.com/mekanism/Mekanism/blob/1.16.x/docs/GoogleStyle4Indents.xml

Reply:
That will certainly help.